src/Controller/Admin/RegisterController.php

```diff
@@ -75,6 +74,6 @@ public function formSave(Request $request, ValidatorInterface $validator)
             ];
         }
 
-        return new Response(json_encode($array));
+        return $this->json($array);
```

Comment:
Hello @korzol

Could you look in details in this controller action and give better name for $array?

Reply:
Hello @ceckoslab 
Now this method returns $result array